java_fibonacci.java

```diff
@@ -0,0 +1,17 @@
+public class Fibonacci {
+
+    public static void main(String[] args) {
+
+        int n = 100, t1 = 0, t2 = 1;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6289366)